strawberryfields/configuration.py

```diff
@@ -158,6 +206,31 @@ def load_config_file(filepath):
     return config_from_file
 
 
+def get_api_section_safely(loaded_config, filepath):
```

Comment:
Since the safety should be implicit, can we rename this `get_api_config`?

Reply:
I'm curious the reason behind adding this function 🙂 I would think that a `KeyError` should be a sufficient warning?

Reply:
Just added it as extra care, as I was trying out the new functionalities had an empty configuration file where `KeyError` was not too descriptive.